### PR TITLE
Rajrang/fix issue1969

### PR DESCRIPTION
**Pull request by rajkumar-rangaraj**
Fix Issue #1969  .

## Changes
(Please provide a brief description of the changes here.)
Handled exception and sending exception message to event source. 

### Checklist
- [X] I ran Unit Tests locally.
- [X] CHANGELOG.md updated with one line description of the fix, and a link to the original issue if available.

### CODE REVIEW

File: LOGGING/src/ILogger/ApplicationInsightsLoggerEventSource.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+namespace Microsoft.Extensions.Logging.ApplicationInsights
+{
+    using System.Diagnostics.Tracing;
+    using System.Reflection;
+
+    /// <summary>
+    /// EventSource for reporting errors and warnings from Logging module.
+    /// </summary>
+    [EventSource(Name = "Microsoft-ApplicationInsights-Logger-EventSourceListener")]
```
Comment (cijothomas) [inline LOGGING/src/ILogger/ApplicationInsightsLoggerEventSource.cs:16]: EventSourceListener is a typo?
I'd call it Microsoft-ApplicationInsightsLoggerProvider or something like that

Comment (cijothomas) [reply LOGGING/src/ILogger/ApplicationInsightsLoggerEventSource.cs:16]: https://github.com/microsoft/ApplicationInsights-Home/tree/master/Samples/ETW can you update it here as well? 
And docs too: https://docs.microsoft.com/en-us/azure/azure-monitor/app/asp-net-troubleshoot-no-data#PerfView

Comment (rajkumar-rangaraj) [reply LOGGING/src/ILogger/ApplicationInsightsLoggerEventSource.cs:16]: Yes, it's  a typo, tried changing from constant to normal on Event source decoration and added listener additionally. 

Good call out to update other articles, was thinking about it. Not sure, Home repo will allow an update as it is in archived state.

File: LOGGING/src/ILogger/ApplicationInsightsLoggerEventSource.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    {
+        public static readonly ApplicationInsightsLoggerEventSource Log = new ApplicationInsightsLoggerEventSource();
+        public readonly string ApplicationName;
+
+        private ApplicationInsightsLoggerEventSource()
+        {
+            this.ApplicationName = GetApplicationName();
+        }
+
+        [Event(1, Message = "Writing an entry to log has failed. Error: {0}", Level = EventLevel.Error)]
```
Comment (cijothomas) [inline LOGGING/src/ILogger/ApplicationInsightsLoggerEventSource.cs:27]: `sending log to ApplicationInsightsLoggerProvider has failed.`

**Pull request merged**